okhttp/src/androidMain/kotlin/okhttp3/OkHttp.android.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2025 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3
+
+import android.content.Context
+import okhttp3.internal.CONST_VERSION
+import okhttp3.internal.platform.Platform
+import okhttp3.internal.platform.PlatformRegistry
+
+actual object OkHttp {
+  @JvmStatic
+  actual val Version: String = CONST_VERSION
```

Comment:
Isn’t `VERSION` sufficient?

Reply:
Not sure how to have expect JvmField.

Reply:
Ooooh, ouch.

Reply:
I think it just works if you put `@JvmField` on the actuals?

```kotlin
expect object OkHttp {
  @JvmStatic
  val VERSION: String
}
```

```kotlin
actual object OkHttp {
  @Suppress("MayBeConstant") // Non-const so external callers get the runtime version.
  @JvmField
  actual val VERSION: String = CONST_VERSION
}
```

Reply:
This has a warning

```
[ACTUAL_ANNOTATIONS_NOT_MATCH_EXPECT] Annotation `@JvmStatic()` is missing on actual declaration.
All annotations from expect 'val VERSION: String get(): String' must be present with the same arguments on actual 'field:val VERSION: String get(): String', otherwise they might behave incorrectly.
```

That sounds ominous.

But seems to work in common test and the API files.

Reply:
https://github.com/square/okhttp/pull/8971

I swear this didn't work for me. But it is now.